addons/popochiu/engine/popochiu.gd

```diff
@@ -3,7 +3,7 @@ extends Node
 ## This is Popochiu's main hub, and is in charge of making the game to work.
 ## 
 ## Is the shortcut for [b]Popochiu.gd[/b], and can be used (from any script) with [b]E[/b] (E.g.
-## [code]E.goto_room("House")[/code]).
+## [code]E.camera.shake()[/code]).
```

Comment:
I think this is an error. Shouldn't it be related to the goto room?

Reply:
Nope. Is just an example of what you can do with **E**.